<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelSchemeWindow</class>
 <widget class="QMainWindow" name="LevelSchemeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>494</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LevelSchemeWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_savePic"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>579</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::RightToLeft</enum>
   </property>
   <widget class="QMenu" name="menuother_isotopes">
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="title">
     <string>other isotopes</string>
    </property>
    <addaction name="actioniso1"/>
    <addaction name="actioniso2"/>
   </widget>
   <widget class="QMenu" name="menuedit_levels">
    <property name="title">
     <string>edit levels</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_savePic">
    <property name="title">
     <string>save picture</string>
    </property>
   </widget>
   <addaction name="menuother_isotopes"/>
   <addaction name="menu_savePic"/>
   <addaction name="menuedit_levels"/>
  </widget>
  <action name="actionss">
   <property name="text">
    <string>ss</string>
   </property>
  </action>
  <action name="actiondd">
   <property name="text">
    <string>dd</string>
   </property>
  </action>
  <action name="actiond">
   <property name="text">
    <string>d</string>
   </property>
  </action>
  <action name="actionds">
   <property name="text">
    <string>ds</string>
   </property>
  </action>
  <action name="actionfaf">
   <property name="text">
    <string>faf</string>
   </property>
  </action>
  <action name="actioniso_1">
   <property name="text">
    <string>iso 1</string>
   </property>
  </action>
  <action name="actioniso_2">
   <property name="text">
    <string>iso 2</string>
   </property>
  </action>
  <action name="actioniso_3">
   <property name="text">
    <string>iso 3</string>
   </property>
  </action>
  <action name="action_savePic">
   <property name="text">
    <string>Save Picture</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actioniso1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>iso1</string>
   </property>
  </action>
  <action name="actioniso2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>iso2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
